app/src/main/java/com/cornellappdev/resell/android/ui/components/global/ResellLoadingCard.kt

```diff
@@ -0,0 +1,93 @@
+package com.cornellappdev.resell.android.ui.components.global
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.times
+import com.cornellappdev.resell.android.ui.theme.Stroke
+import com.cornellappdev.resell.android.ui.theme.Wash
+
+@Composable
+fun ResellLoadingCard(
+    modifier: Modifier = Modifier,
+    small: Boolean
+) {
+    val screenWidth = LocalConfiguration.current.screenWidthDp.dp
+    val width = 0.5f * screenWidth
+    val scale = 1.0 / 173.0 * width
+    val height = if (small) 144.73 * scale else 246.4 * scale
+    val contentHeight = if(small) 111.73*scale else 213.4 * scale
+    Column(
+        modifier = modifier
+            .widthIn(max = width)
+            .fillMaxWidth()
+            .clip(RoundedCornerShape(size = 8.dp))
+            .background(Color.White)
+            .border(width = 1.dp, color = Stroke, shape = RoundedCornerShape(8.dp))
+            .heightIn(max = height)
+    ) {
+        Box(
+            modifier = modifier
+                .heightIn(max = contentHeight)
+                .fillMaxSize()
+                .background(
+                    Wash
+                )
```

Comment:
We want the loading screens to pulse like the loading URL card does:

https://github.com/user-attachments/assets/ae059aee-3524-4ebc-89c7-77eb5cc44ceb

You can achieve this using the `.shimmer()` modifier I made. Check `ResellCard.kt` for an example of this!

Reply:
Oh and you can test this by going into interactive mode in your preview:

![image](https://github.com/user-attachments/assets/57d1bc98-70f7-4a6f-aa81-22fe21d2e5ad)